package/yast2-packager.changes

```diff
@@ -3,6 +3,53 @@ Fri Mar  3 16:42:50 UTC 2017 - ancor@suse.com
 
 - Synced storage-ng and master branches. See changes below.
 - 3.3.3
+-------------------------------------------------------------------
+Fri Apr  7 13:40:59 UTC 2017 - lslezak@suse.cz
+
+- Create the /etc/products.d/baseproduct symlink in inst-sys when
+  it is missing to allow expanding variables in the add-on
+  repositories URL (bsc#972046)
+- 3.2.22
```

Comment:
When merging with master, we usually remove these 3.2.X version numbers from the changelog. Those versions never existed in this branch/OBS project.

Pretty much the same we always do when merging fixes done in a maintenance branch (let's say 3.1.x) into a newer branch (let's say master with 3.2.x).